app/cdap/components/SourceControlManagement/helpers.ts

```diff
@@ -123,3 +143,22 @@ export const compareTimeInstant = (t1: ITimeInstant, t2: ITimeInstant): number =
 export const getOperationStartTime = (operation: IOperationRun): string => {
   return moment(operation.metadata?.createTime.seconds * 1000).format('DD-MM-YYYY HH:mm:ss A');
 };
+
+export const getOperationRunTime = (operation: IOperationRun): string => {
+  if (operation.metadata?.createTime && operation.metadata?.endTime) {
+    return moment
+      .duration(
+        (operation.metadata?.endTime.seconds - operation.metadata?.createTime.seconds) * 1000
+      )
+      .humanize();
+  }
+  return null;
+};
+
+export const timeInstantToString = (t?: ITimeInstant): string => {
+  if (!t) {
+    return '--';
+  }
+
+  return moment(t.seconds * 1000).format('DD-MM-YYYY HH:mm:ss A');
```

Comment:
For future consideration: consolidate with date formatting logic in https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/services/DataFormatter/index.ts

Reply:
done.